models/profiles-ml.yaml

```diff
@@ -42,7 +42,6 @@ models:
               features:
                 - name: *percentile_name_7_days
                   description: 'Percentile of churn score. Higher the percentile, higher the probability of churn'
-      !!merge <<: *feature_meta_data_7_days
```

Comment:
`!!merge <<: *feature_meta_data_7_days` should instead be 
`<<: *feature_meta_data_7_days`
It should not be removed completely